build-tools/packages/build-cli/src/handlers/checkFunctions.ts

```diff
@@ -241,30 +241,24 @@ export const checkInstallBuildTools: StateHandlerFunction = async (
 ): Promise<boolean> => {
 	if (testMode) return true;
 
-	const { context } = data;
+	const { context, releaseGroup } = data;
 	assert(context !== undefined, "Context is undefined.");
+	assert(releaseGroup !== undefined, "Release group is undefined.");
```

Comment:
I see that `FluidReleaseStateHandlerData`'s members are all optional.  Should some of them be mandatory?

Reply:
Good point. Refactoring this was easier than I expected.